superset/key_value/commands/update.py

```diff
@@ -16,30 +16,25 @@
 # under the License.
 import logging
 from abc import ABC, abstractmethod
-from typing import Optional
 
-from flask_appbuilder.models.sqla import Model
-from flask_appbuilder.security.sqla.models import User
 from sqlalchemy.exc import SQLAlchemyError
 
 from superset.commands.base import BaseCommand
 from superset.key_value.commands.exceptions import KeyValueUpdateFailedError
+from superset.key_value.commands.parameters import CommandParameters
 
 logger = logging.getLogger(__name__)
 
 
 class UpdateKeyValueCommand(BaseCommand, ABC):
     def __init__(
-        self, actor: User, resource_id: int, key: str, value: str,
+        self, cmd_params: CommandParameters,
     ):
-        self._actor = actor
-        self._resource_id = resource_id
-        self._key = key
-        self._value = value
+        self._parameters = cmd_params
```

Comment:
Could we have the same variable name here? Either `self._cmd_params = cmd_params` or rename in the sig to get `self._parameters = parameters`